lib/OrdinaryDiffEqFeagin/src/alg_utils.jl

```diff
@@ -3,4 +3,4 @@ alg_order(alg::Feagin12) = 12
 alg_order(alg::Feagin14) = 14
 
 alg_adaptive_order(alg::Feagin10) = 8
-alg_adaptive_order(alg::Feagin14) = 12
\ No newline at end of file
+alg_adaptive_order(alg::Feagin14) = 12
```

Comment:
don't reformat all of the other ones. Your formatting here is incorrect. It's already corrected on master.